### PR TITLE
[s] Fixes an exploit allowing admins to read any file

### PR DESCRIPTION
**Pull request by Cyberboss**
I checked all other calls to `file()` they are safe

### CODE REVIEW

File: code/modules/tooltip/tooltip.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@ Notes:
 /datum/tooltip/New(client/C)
 	if (C)
 		src.owner = C
-		src.owner << browse(file2text(src.file), "window=[src.control]")
```
Comment (AnturK) [inline code/modules/tooltip/tooltip.dm]: You could just make this one static

**Comment by AnturK** [comment]
Also [s] on admin-only exploits seems silly.

**Comment by ShadowDeath6** [comment]
can't trust em

**Comment by PKPenguin321** [comment]
Too late as we speak I'm tapping into the mainframe

**Comment by Iamgoofball** [comment]
@AnturK The reason is because if someone ever figures out how to escalate permissions with SS13, they could use these exploits to do bad shit. Our admins are also an entirely volunteer backed force and are therefore prone to going rogue at any point in time.

**Comment by PKPenguin321** [comment]
*hacker voice* I'm in

**Pull request merged**